providers/cncf/kubernetes/src/airflow/providers/cncf/kubernetes/operators/pod.py

```diff
@@ -1132,7 +1132,10 @@ def _build_find_pod_label_selector(self, context: Context | None = None, *, excl
             **self.labels,
```

Comment:
My immediate instinct is to not include user provided labels in the first place! But, I'm a [bit late to that objection](https://github.com/apache/airflow/pull/33057).

Reply:
Yeah, similar qn. But alright...